python/cudf/cudf/core/column/string.py

```diff
@@ -4861,6 +4861,11 @@ def find_last_value(self, value, closest=False):
         return self._find_first_and_last(value)[1]
 
     def normalize_binop_value(self, other):
+        # fastpath: gpu scalar
+        if isinstance(other, cudf.Scalar) and other.dtype == "object":
+            return libcudf.column.make_column_from_scalar(
```

Comment:
See my comment about `make_column_from_scalar` at the end -- ditto for anywhere else we are calling it.

Reply:
Done 👍